helloworld-springboot/README.md

```diff
@@ -64,6 +64,16 @@ gcloud components update app-engine-java
 gcloud components update
 ```
 
+### Configure the `app.yaml` descriptor
+
+The [`app.yaml`][app-yaml] descriptor is optional, and is used to describe
+URL dispatch and resource requirements.  We recommend that you initially
+configure [`manual_scaling`][manual-scaling] and disable 
+[health-checks][health-checking].
```

Comment:
I'm not sure if we want to say we should disable health checking as a recommendation. We should update `app.yaml` to comment-out the health-checking with a note to enable it for production use.